src/ir_LG.cpp

```diff
@@ -389,10 +384,10 @@ void IRLgAc::setMode(const uint8_t mode) {
     case kLgAcHeat:
     case kLgAcCool:
     case kLgAcFan:
-      setBits(&remote_state, kLgAcModeOffset, kLgAcModeSize, mode);
+      _.Mode = mode;
       break;
-    default:  // If we get an unexpected mode, default to AUTO.
```

Comment:
Any particular reason for this comment being removed?

Reply:
I don't think it's necessary